server/controllers/inventory/inventory/core.js

```diff
@@ -216,3 +266,15 @@ function errorHandler(error, req, res, next) {
     next(error);
   }
 }
+
+function inventoryLog(inventoryUuid) {
+  const sql = `
+    SELECT ivl.text, ivl.log_timestamp, user.display_name as userName
+    FROM inventory_log ivl
+    JOIN user on user.id = ivl.user_id
+    WHERE ivl.inventory_uuid=?
+    ORDER BY ivl.log_timestamp ASC
```

Comment:
Could you order by DESC this query, it will be more useful to see last change first.